lib/cisco_node_utils/cmd_ref/fabricpath_topology.yaml

```diff
@@ -0,0 +1,35 @@
+# fabricpath_topology
+---
+# Fabricpath feature is not available on N3K and N9K
+_exclude: [/N3/, /N9K/]
+
+all_topos:
+  multiple:
+  config_get: "show fabricpath topology"
```

Comment:
I don't see that as a problem. If the configuration fails to match the regex then the property is not configured. If it's not configured then it is using the default configuration.